site/docs/types/type_variations.md

```diff
@@ -0,0 +1,11 @@
+# Type Variations
+
+Since Substrait is designed to work in both logical and physical contexts, there is a need to support extended attributes in the physical context. Different consumers may have multiple ways to present the same logical type. For example, an engine might support dictionary encoding a string or using either a row-wise or columnar representation of a struct. As such, there is the facility for specification users to express additional type variations for each logical type. These variations are expected to have the same logical properties as the canonical variation and are defined for each organization. The key properties of these variations are:
+
+| Property          | Description                                                  |
+| ----------------- | ------------------------------------------------------------ |
+| Parent Type       | The parent type this variation belongs to. Variations can only be expressed for simple types and wild-carded compound types (e.g. i8 or varchar(*)). |
```

Comment:
Maybe `Logical Type` instead of `Parent Type`?  When I see `Parent` I think tree but there is only one level of nesting here.  That being said, I don't feel too strongly about it.

Reply:
I'm trying to avoid using logical since it is overloaded in relational algebra. Maybe "base" instead?

Reply:
updated to base. lmk your thoughts.

Reply:
Base works for me